### ISSUE TITLE
Script needed to remove "Overdrive" from subject tags

### ISSUE THREAD

**Issue by seabelis** [internetarchive/openlibrary#2106]
## Is your feature request related to a problem? Please describe.
"Overdrive" appears frequently in subject tags but is no longer needed as OL does not use Overdrive for lending.

## Describe the solution you'd like
I can remove these manually as I encounter them, but a script to eliminate all of them would be appreciated.

**Comment by tfmorris** [comment]
We should identify and remove all the non-subject "subjects" in a single pass rather than doing them one at a time.

I know we've discussed this in the past, but don't immediately see a relevant issue.

**Comment by brad2014** [comment]
Any objection to retitling this issue "Remove Overdrive and other non-subject subject tags"

**Comment by cdrini** [comment]
Yes; let's keep this specific to overdrive for now. If we expand to "other non-subject subject tags" it'll take forever to complete and require investigation and community consensus.

**Comment by tfmorris** [comment]
Totals 66K works https://openlibrary.org/subjects/overdrive

We should also remove the `id_overdrive` field from the Solr schema and any uses of it.

**Comment by tfmorris** [comment]
I've created #2107 for the removal of the rest of the fake subjects (which are much more frequent than `Overdrive`)

**Comment by LeadSongDog** [comment]
Hang on a bit. Is the overdrive tag and identifier not useful to users borrowing the edition from other libraries? Most books with isbns are not available for OL loans, but we certainly keep those identifiers.

**Comment by hornc** [comment]
The Overdrive subject was only present on works, not editions, and there was no specific id for any I found. If we did want to link them, `identifiers` would be the appropriate field. For other library borrowable copies we provide links to World Cat and Library Link. I'm not sure if these overdrive tags ever did anything. It looks like they were added when some description and other metadata was added at some point in the past.
